<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Assets</class>
 <widget class="QMainWindow" name="Assets">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>1000</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>420</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Implicit Terrain</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QGroupBox" name="Objects_groupBox">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>350</width>
        <height>250</height>
       </size>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QPushButton" name="resetcameraButton">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>101</width>
         <height>23</height>
        </rect>
       </property>
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#5500ff;&quot;&gt;Callback #03&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-style:italic; color:#aa00ff;&quot;&gt;Create Eurographics 1996 Bird&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="text">
        <string>Reset Camera</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
       <property name="default">
        <bool>false</bool>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QGroupBox" name="groupBox">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>250</y>
         <width>321</width>
         <height>201</height>
        </rect>
       </property>
       <property name="title">
        <string>Implicit Shapes</string>
       </property>
       <widget class="QPushButton" name="sphereImplicit">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>40</y>
          <width>91</width>
          <height>23</height>
         </rect>
        </property>
        <property name="toolTip">
         <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; color:#5500ff;&quot;&gt;Callback #01&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-style:italic; color:#aa00ff;&quot;&gt;Create Eurographics 1996 Bear&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="text">
         <string>Sphere</string>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
       </widget>
       <widget class="QPushButton" name="torusImplicit">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>80</y>
          <width>89</width>
          <height>25</height>
         </rect>
        </property>
        <property name="text">
         <string>Torus</string>
        </property>
       </widget>
       <widget class="QPushButton" name="boxImplicit">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>80</y>
          <width>89</width>
          <height>25</height>
         </rect>
        </property>
        <property name="text">
         <string>Box</string>
        </property>
       </widget>
       <widget class="QPushButton" name="capsuleImplicit">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>40</y>
          <width>89</width>
          <height>25</height>
         </rect>
        </property>
        <property name="text">
         <string>Capsule</string>
        </property>
       </widget>
       <widget class="QPushButton" name="unionImplicit">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>140</y>
          <width>89</width>
          <height>25</height>
         </rect>
        </property>
        <property name="text">
         <string>Union</string>
        </property>
       </widget>
       <widget class="QPushButton" name="intersectionImplicit">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>140</y>
          <width>89</width>
          <height>25</height>
         </rect>
        </property>
        <property name="text">
         <string>Intersection</string>
        </property>
       </widget>
       <widget class="QPushButton" name="melangeImplicit">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>120</y>
          <width>89</width>
          <height>25</height>
         </rect>
        </property>
        <property name="text">
         <string>Melange</string>
        </property>
       </widget>
       <widget class="QPushButton" name="subtractionImplicit">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>160</y>
          <width>89</width>
          <height>25</height>
         </rect>
        </property>
        <property name="text">
         <string>Subtraction</string>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="groupBox_4">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>110</y>
         <width>171</width>
         <height>121</height>
        </rect>
       </property>
       <property name="title">
        <string> Statistics</string>
       </property>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>30</y>
          <width>61</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Vertex</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_2">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>60</y>
          <width>81</width>
          <height>20</height>
         </rect>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>60</y>
          <width>61</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Triangles</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>30</y>
          <width>81</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="groupBox_13">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>110</y>
         <width>151</width>
         <height>121</height>
        </rect>
       </property>
       <property name="title">
        <string>Shading</string>
       </property>
       <widget class="QRadioButton" name="radioShadingButton_1">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>30</y>
          <width>91</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Normal</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QCheckBox" name="wireframe">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>90</y>
          <width>101</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Wireframe</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QRadioButton" name="radioShadingButton_2">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>60</y>
          <width>61</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Color</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>470</y>
         <width>321</width>
         <height>171</height>
        </rect>
       </property>
       <property name="title">
        <string>Mesh Shapes</string>
       </property>
       <widget class="QPushButton" name="boxMesh">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>40</y>
          <width>101</width>
          <height>21</height>
         </rect>
        </property>
        <property name="toolTip">
         <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; color:#5500ff;&quot;&gt;Callback #01&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-style:italic; color:#aa00ff;&quot;&gt;Create Eurographics 1996 Bear&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="text">
         <string>Box</string>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget_GL" native="true"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Esc</string>
   </property>
  </action>
  <action name="actionNew_Scene">
   <property name="text">
    <string>New Scene</string>
   </property>
  </action>
  <action name="actionExport_obj">
   <property name="text">
    <string>OBJ File (.obj)</string>
   </property>
  </action>
  <action name="actionExport_ma">
   <property name="text">
    <string>Maya File (.ma)</string>
   </property>
  </action>
  <action name="actionLoad_ma">
   <property name="text">
    <string>Load</string>
   </property>
  </action>
  <action name="actionGL">
   <property name="text">
    <string>GL</string>
   </property>
  </action>
  <action name="actionVectorized">
   <property name="text">
    <string>Vectorized</string>
   </property>
  </action>
  <action name="actionExport_svg">
   <property name="text">
    <string>SVG File (.svg)</string>
   </property>
  </action>
  <action name="actionSphere_Tracing_png">
   <property name="text">
    <string>Sphere Tracing (.png)</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
